.cypress/integration/7_app_analytics.spec.js

```diff
@@ -159,25 +162,31 @@ describe('Creating application', () => {
   });
 
   it('Saves time range for each application', () => {
+    cy.wait(delay);//Fail to load without
     cy.get('[data-test-subj="nameFormRow"]').type(nameTwo);
     cy.get('[data-test-subj="logSourceAccordion"]').trigger('mouseover').click();
-    cy.get('[data-test-subj="searchAutocompleteTextArea"]').focus().type(baseQuery, {delay: TYPING_DELAY});
+    cy.get('[data-test-subj="searchAutocompleteTextArea"]').focus().type('      ' +baseQuery);
     cy.get('[data-test-subj="createButton"]').should('not.be.disabled');
     cy.get('[data-test-subj="createButton"]').click();
+    cy.get('.euiTableRow').should('have.length.lessThan', 1);
     cy.get('[data-test-subj="applicationTitle"]').should('contain', nameTwo);
     changeTimeTo24('weeks');
     cy.get('[data-test-subj="superDatePickerShowDatesButton"]').should('contain', 'Last 24 weeks');
-    cy.get('.euiBreadcrumb[href="#/application_analytics"]').click();
-    cy.wait(delay);
+    cy.get('.euiBreadcrumb[href="#/"]').click();
+    cy.get('.euiTableRow').should('have.length.greaterThan', 1);
     cy.get(`[data-test-subj="${nameOne}ApplicationLink"]`).click();
+    cy.wait(delay*3);
+    cy.get('.euiTableRow').should('have.length.lessThan', 1);
     cy.get('[data-test-subj="applicationTitle"]').should('contain', nameOne);
     changeTimeTo24('months');
     cy.get('[data-test-subj="superDatePickerShowDatesButton"]').should('contain', 'Last 24 months');
-    cy.get('.euiBreadcrumb[href="#/application_analytics"]').click();
+    cy.get('.euiBreadcrumb[href="#/"]').click();
+    cy.get('.euiTableRow').should('have.length.greaterThan', 1);
     cy.get(`[data-test-subj="${nameTwo}ApplicationLink"]`).click();
+    cy.get('.euiTableRow').should('have.length.lessThan', 1);
```

Comment:
replace with 0 or something more clear?

Reply:
<img width="394" alt="Screenshot 2023-07-17 at 10 57 17 AM" src="https://github.com/opensearch-project/dashboards-observability/assets/105462877/32c9248f-ae23-4a9d-9fa9-a30384b2b50b">

Other objects on the new page are considered table but have nothing in them so below 0 doesn't work.